caffe2/proto/torch.proto

```diff
@@ -4,6 +4,46 @@ import "caffe2/proto/caffe2.proto";
 
 package torch;
 
+message TensorDef {
+  // the tensor id in ModelDef's tensor table
+  optional string tensor_id = 1;
+  repeated int64 dims = 2;
+  optional int64 offset = 3;
+  repeated int64 strides = 4;
+
+  // the same as caffe2.TensorProto.DataType
```

Comment:
can we use the type directly? (I think enum - int32 proto change is backward compatible: https://groups.google.com/forum/#!topic/protobuf/iNIRHPXjAds )